docs/source/supergraphs.md

```diff
@@ -43,7 +43,27 @@ subgraphs:
       file: ./people.graphql
 ```
 
-The YAML file must specify each subgraph's public-facing URL (`routing_url`), along with the path to its schema (`schema.file`).
+In the above example, The YAML file specifies each subgraph's public-facing URL (`routing_url`), along with the path to its schema (`schema.file`).
+
+It's also possible to pull subgraphs from various sources and specify them in the YAML file. For example, here is a configuration that specifies schema using Apollo Registry refs (`subgraph`, `graphref`), introspection(`url`) and subgraph introspection (`subgraph_url`):
```

Comment:
For the same reasons I noted above.  (Though I won't add `file` here since it's noted on the line above).

```suggestion
It's also possible to pull subgraphs from various sources and specify them in the YAML file. For example, here is a configuration that specifies schema using Apollo Registry refs (`subgraph`, `graphref`) and subgraph introspection (`subgraph_url`):
```